keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -736,7 +736,7 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 #### Beta
 - Metrics are added to the kube-apiserver (see the [monitoring-requirements](#monitoring-requirements) section for more details)
 - Implement `SendInitialEvents` for `watch` requests in the etcd storage implementation
-- The feature is enabled for kube-apiserver and kube-controller-manager 
+- The feature is enabled for kube-apiserver, kube-controller-manager and kubelet
```

Comment:
alternatively we could enable for the kube-scheduler or for both, i don't have a strong opinion.

Reply:
We're already in beta without kubelet. So let's maybe make it explicit that we're talking about "second beta" or sth like that.

Reply:
updated, ptal.